red-hat-camel-k/INFO.md

```diff
@@ -0,0 +1,46 @@
+# red-hat-camel-k
+
+Red Hat Integration - Camel K
+==============
+
+**Red Hat Integration - Camel K** is a lightweight integration platform, born on Kubernetes, with serverless superpowers.
+
+## Installation
+
+To start using Camel K, install the operator and then create the following `IntegrationPlatform`:
+```
+apiVersion: camel.apache.org/v1
+kind: IntegrationPlatform
+metadata:
+  name: camel-k
+  labels:
+    app: "camel-k"
+```
+
+An `IntegrationPlatform` resource is automatically created by default, so you can skip this step.
+Also, You can edit the `IntegrationPlatform`, to configure Camel K.
+The configuration from the `IntegrationPlatform` will apply to the Camel integrations created in the same project.
+
+## Running an Integration
+
+After the initial setup, you can run a Camel integration on the cluster by creating an example `Integration`, e.g.:
+```
+apiVersion: camel.apache.org/v1
+kind: Integration
+metadata:
+  name: example
+spec:
+  sources:
+  - name: Example.java
```

Comment:
Same as the other comment, this documentation was generated by the Operator metadata / docs itself, perhaps we should improve that first ?

Another option is to remove `INFO.md` completely, so that we don't enter into the details here as it's not the best place (and it will probably get outdated eventually)

Reply:
@christophd any thoughts here?

Reply:
@pierDipi ah ok, agreed thanks! in case this is generated from the Operator metadata this PR is not the right place to fix but the Operator metadata itself. We can leave `INFO.md` as it is and update as soon as the Operator metadata has improved. It is not a big issue and the information given is not wrong. I was just wondering whether YAML would be a better fit here